packages/genesys-spark-components/src/components/beta/gux-label-info/gux-label-info.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, Element, h, JSX, Prop } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxLabelInfoVariant } from './gux-label-info.types';
+
+/**
+ * @slot content
+ */
+
+@Component({
+  tag: 'gux-label-info-beta',
+  shadow: true
+})
+export class GuxLabelInfo {
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  variant: GuxLabelInfoVariant = 'info';
+
+  componentWillLoad(): void {
+    trackComponent(this.root, { variant: this.variant });
+  }
+
+  private getVariantIcon(variant: GuxLabelInfoVariant): string {
+    switch (variant) {
+      case 'question':
+        return 'fa/circle-question-regular';
+      default:
+        return 'fa/circle-info-regular';
+    }
+  }
+
+  render(): JSX.Element {
+    return (
+      <gux-icon-tooltip-beta icon-name={this.getVariantIcon(this.variant)}>
```

Comment:
I was thinking we could set the variant icon once instead of calling this function every time the component renders.